packages/app/app/studio/[organization]/event/components/stageSettings/stagesAccordion.tsx

```diff
@@ -18,11 +28,28 @@ const StageAccordion = ({ stages }: { stages: IStageModel[] }) => {
     setSelectedSetting,
   } = useNavigation()
 
-  const handleDeleteStage = (stageId: string, streamId: string) => {
+  const handleDeleteStage = async (
+    stageId: string,
+    organizationId: string,
+    streamId: string
+  ) => {
     if (
       window.confirm('Are you sure you want to delete this stage?')
```

Comment:
I think `window.confirm` is a quite bad UX. Is it possible to show a Modal out of this?

Reply:
Does not have high-prio though rn

Reply:
https://ui.shadcn.com/docs/components/alert-dialog